tests/test_player_with_strategy_one.py

```diff
@@ -1,26 +1,73 @@
-                    ['blue', 'blue',  'red', 'blue',  'red', 'blue', None],
-                    ['red',   'red',  'red', 'blue',  'red', 'blue', None]])
-        self.match = Match(self.player_blue, self.player_red, self.board_player_blue_wins)
-        self.assertFalse(self.player_blue.is_winner(self.board_player_blue_wins))
-        for i in range(8):
-            self.match.play_next_turn()
-            self.assertFalse(self.player_blue.is_winner(self.board_player_blue_wins))
-        self.match.play_next_turn()
-        self.assertTrue(self.player_blue.is_winner(self.board_player_blue_wins))
+    def test_player_with_strategy_one_blue_wins_on_right_upper_corner(self):
+        board = GameBoard.from_matrix([
+                    ['red',  'blue',  'red',  'red', 'blue',  'blue',   None],
+                    ['red',   'red', 'blue',  'red',  'red',  'red', 'blue'],
+                    ['red',  'blue',  'red',  'red', 'blue', 'blue', 'blue'],
+                    ['blue', 'blue',  'red', 'blue',  'red', 'blue', 'blue'],
+                    ['red',  'blue',  'red', 'blue',  'red', 'blue',  'red'],
+                    ['blue',  'red', 'blue',  'red',  'red',  'red',  'red']])
+        self.assertEqual(board.read_entry(6, 7), None)
+        self.assertFalse(self.player_blue.is_winner(board))
+        self.player_blue.play(board)
+        self.assertEqual(board.read_entry(6, 7), 'blue')
+        self.assertTrue(self.player_blue.is_winner(board))
+
+    def test_player_with_strategy_one_blue_wins_on_left_corner(self):
+        board = GameBoard.from_matrix([
+                    [None,    None,    None,  None, 'blue',  'blue',   None],
+                    ['blue',   None,   None,  'red',  'red',  'red', 'blue'],
+                    ['blue', 'blue',   None,  'red', 'blue', 'blue', 'blue'],
+                    ['blue', 'blue',  'red', 'blue',  'red', 'blue', 'blue'],
+                    ['red',  'blue',  'red', 'blue',  'red', 'blue',  'red'],
+                    ['blue',  'red', 'blue',  'red',  'red',  'red',  'red']])
+        self.assertEqual(board.read_entry(6, 1), None)
+        self.assertFalse(self.player_blue.is_winner(board))
+        self.player_blue.play(board)
+        self.assertEqual(board.read_entry(6, 1), 'blue')
+        self.assertTrue(self.player_blue.is_winner(board))
+
+    def test_player_with_strategy_one_red_not_winning_on_column(self):
+        board = GameBoard.from_matrix([
+                    [None,     None,   None,   None,  None, None, None],
+                    [None,     None,   None,   None,  None, None, None],
+                    ['red',  'blue',   None, 'blue',  None, None, None],
+                    ['blue', 'blue',   None, 'blue',  None, None, None],
+                    ['red',  'blue',  'red', 'blue',  None, None, None],
+                    ['blue',  'red', 'blue',  'red', 'red', None, None]])
+        self.assertEqual(board.read_entry(5, 1), None)
```

Comment:
Why are you testing on three locations here and in the previous tests in only one? You should be consistent on the way you test, otherwise it gets confusing.